src/TogglePassword/doc/index.rst

```diff
@@ -0,0 +1,216 @@
+Symfony UX TogglePassword
+=========================
+
+Symfony UX TogglePassword is a Symfony bundle providing visibility toggle for password inputs
+in Symfony Forms. It is part of `the Symfony UX initiative`_.
+
+It allows visitors to switch the type of password field to text and vice versa.
+
+Installation
+------------
+
+Before you start, make sure you have `StimulusBundle configured in your app`_.
+
+Then, install this bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-toggle-password
+
+If you're using WebpackEncore, install your assets and restart Encore (not
+needed if you're using AssetMapper):
+
+.. code-block:: terminal
+
+    $ npm install --force
+    $ npm run watch
+
+    # or use yarn
+    $ yarn install --force
+    $ yarn watch
+
+Usage
+-----
+
+Any ``PasswordType`` can be transformed into a toggle password field by adding the ``toggle`` option::
+
+    // ...
+    use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+
+    class CredentialFormType extends AbstractType
+    {
+        public function buildForm(FormBuilderInterface $builder, array $options)
+        {
+            $builder
+                // ...
+                ->add('password', PasswordType::class, ['toggle' => true])
+                // ...
+            ;
+        }
+
+        // ...
+    }
+
+Customizing labels and icons
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony UX TogglePassword uses English by default and SVG icons from `Heroicons`_ for the toggle button.
```

Comment:
```suggestion
The field uses the words "Show" and "Hide" by default and SVG icons from `Heroicons`_ for the toggle button.
```

I'm removing the part about English because if you need to translate, the more important thing is setting up the translations, which is mentioned below.